es3.0/bottom.php

```diff
@@ -1,9 +1,22 @@
 <html>
 <head>
 <title>OpenGL ES 3.0 Reference Pages</title>
+  <script>
+    window.onload = function () {
+      try {
+        let params = new URL(document.location.toString()).searchParams;
```

Comment:
2. This is the second place it hits and then sets the query params on the subpage